scripts/convert_to_csv.py

```diff
@@ -1,40 +1,35 @@
 import pyreadr
-import glob, os
+from pathlib import Path
 
 
 errors = []
+need_zip = [
+    "ncdc_normals.rda",
+]

```

Comment:
It's purely conventional, but a strong convention in Python to make "globals" like this all caps, i.e. `ERRORS` and `NEEDS_ZIP`.  

I'd also suggest either a tuple or a set for `NEEDS_ZIP`. It's not a big issue in a small script, but in general those are good practice. A tuple would be immutable and it's always advised to use immutable data structures if there is not an explicit need for mutability. Alternatively a set offers efficient constant-time lookup so it's good for inclusion testing (tho in this case I think since the size is very small, I think I'd go with the tuple)